transport/src/main/java/io/netty/channel/SingleThreadEventLoop.java

```diff
@@ -91,14 +91,10 @@ public ChannelFuture register(final ChannelPromise promise) {
     @Deprecated
     @Override
     public ChannelFuture register(final Channel channel, final ChannelPromise promise) {
-        if (channel == null) {
-            throw new NullPointerException("channel");
-        }
-        if (promise == null) {
-            throw new NullPointerException("promise");
-        }
-
-        channel.unsafe().register(this, promise);
+        ObjectUtil.checkNotNull(promise, "promise");
+        ObjectUtil.checkNotNull(channel, "channel").
+                unsafe().
+                register(this, promise);
```

Comment:
nit consider merging some lines... The formatting looks strange.

Reply:
use this version. this looks better..

```java
ObjectUtil.checkNotNull(promise, "promise");
ObjectUtil.checkNotNull(channel, "channel");
channel.unsafe().register(this, promise);
```